bll/Services/AccountService.cs

```diff
@@ -0,0 +1,101 @@
+﻿using AutoMapper;
+using bll.DTO.User;
+using bll.Interfaces;
+using dal.Context;
+using dal.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace bll.Services
+{
+    public class AccountService : IAccountService
+    {
+        private readonly UserManager<User> userManager;
+        private readonly RoleManager<ApplicationRole> roleManager;
+        private readonly IConfiguration configuration;
+        private readonly IMapper mapper;
+
+        public AccountService(UserManager<User> userManager, RoleManager<ApplicationRole> roleManager, IConfiguration configuration, IMapper mapper)
+        {
+            this.configuration = configuration;
+            this.userManager = userManager;
+            this.roleManager = roleManager;
+            this.mapper = mapper;
+        }
+
+        public async Task<string> Login(LoginModel userLogin)
+        {
+            var user = await userManager.FindByNameAsync(userLogin.UserName);
+            var checkPassword = await userManager.CheckPasswordAsync(user, userLogin.Password);
+
+            if (user == null) throw new NullReferenceException();
+            if (!checkPassword) throw new Exception("Wrong password");
+            if (DateTime.UtcNow < user.LockoutEnd) throw new Exception("User is blocked");
```

Comment:
why don't you create a custom exception for this case? You also can introduce some enum as an error code and provide it as standardized exception reason that is not dependent on localization.